### ISSUE TITLE
MAML Inductive setting

### ISSUE THREAD

**Issue by oveilleux** [fmu2/PyTorch-MAML#6]
Hi, I think bringing up the inductive bias of MAML method is very interesting. How exactly could we get the results of MAML in the inductive setting using your implementation? 

Should we be setting episodic variable to True of BatchNorm2d class https://github.com/fmu2/PyTorch-MAML/blob/19246a1ff9a1f47470dce80ed9e9332042f9e151/models/modules.py#L96-L99 during meta-testing phase. That way, the normalization statistics would be based on a single task and not the mini-batch. 

Thank you.
